packages/mdc-select/mdc-select.scss

```diff
@@ -66,6 +66,11 @@
       display: none;
     }
 
+    &::-ms-value {
+      background-color: transparent;
+      color: inherit;
```

Comment:
Would replacing this with `@include mdc-select-ink-color($mdc-select-ink-color);` obviate the need for #3513?